airflow/www/templates/airflow/dags.html

```diff
@@ -57,7 +57,8 @@
       Airflow found incompatible data in the <code>{{ original_table_name }}</code> table in the
       metadatabase, and has moved them to <code>{{ moved_table_name }}</code> during the database migration
       to upgrade. Please inspect the moved data to decide whether you need to keep them, and manually drop
-      the <code>{{ moved_table_name }}</code> table to dismiss this warning.
+      the <code>{{ moved_table_name }}</code> table to dismiss this warning.<p/>
```

Comment:
This looks unintended?

Reply:
Intended actually - I added <p> so that the link is below the message.

Reply:
If the intention is to split the two parts, I'm not sure this is a recommended way.

Reply:
Not really - more to have the link very close on the right side rather than in a very unspecified place depending on the screen size:

![Screenshot from 2021-11-07 20-16-08](https://user-images.githubusercontent.com/595491/140658665-b8c5c3ee-f2f6-4c15-931c-71499c72a7f9.png)

Other suggestions how it can be done better if not via paragraph?